paasta_tools/mesos_tools.py

```diff
@@ -615,3 +615,36 @@ def get_mesos_network_for_net(net):
         'host': 'HOST',
     }
     return docker_mesos_net_mapping.get(net, net)
+
+
+def get_mesos_task_count_by_slave(mesos_state, pool='default'):
+    """Get counts of running tasks per mesos slave. Also include separate count of chronos tasks
+
+    :param mesos_state: mesos state dict
+    :param pool: pool of slaves to return (None means all)
+    :returns: dict of {<slave_id>:{'count': <count>, 'slave': <slave_object>, 'chronos_count':<chronos_count>}}
+    """
+    all_mesos_tasks = get_running_tasks_from_active_frameworks('')  # empty string matches all app ids
+    if pool:
```

Comment:
I put this in so that there's a way to get *all* the pools. I'm not using it for the autoscaler but I think I'll leave it as it could be useful elsewhere.